daml/Marketplace/Clearing/Market/Model.daml

```diff
@@ -0,0 +1,51 @@
+module Marketplace.Clearing.Market.Model where
+
+import DA.Finance.Types
+import DA.Next.Set (Set)
+
+template FairValueCalculationRequest
+  with
+    operator : Party
+    provider : Party
+    customer : Party
+    optListingIds : Optional [Text]
+    currency : Id
+    upTo : Time
+  where
+    signatory provider
+    controller provider can
+      FairValueCalculationRequest_Ack : ()
+        do return ()
+
+template ManualFairValueCalculation
+  with
+    provider : Party
+    customer : Party
+    listingId : Text
+    currency : Id
+    upTo : Time
+    observers : Set Party
+  where
+    signatory provider
+
+    controller customer can
```

Comment:
Doesn't make sense to me from a business point of view that the customer is valuing their own positions... Maybe related to @brian-weir 's comment above.

Reply:
The customer here is the listing agent, not a clearinghouse member, this is the same `Service` that the listing agent uses to request the clearinghouse cover their market.  Does it make more sense to you that way? It used to be the exchange handling any manual valuations, and since that is now split out into multiple services, the listing part of it made the most sense to me.

In the end I think this will fit best somewhere in the listing service directly, so any customer can get fair values of instruments (regardless of if they are cleared or not).

Reply:
I would avoid using 'customer' as it can mean many things. Typically it would not be a 3rd party with no affiliations. Does either 'issuer' or 'exchange' fit the description?

Reply:
I was going with `customer` to match the service but I would prefer `exchange` anyway.

Reply:
Almost all our services have `operator, provider, customer` parties as part of their template and also the template's signature. It makes it clear which party is the producer and which is the consumer of said services. Plus, once we get generics (or what they are calling 'interfaces') back into Daml, it should (in theory...) be easy to extract these details out of all our services (along with the offer/request templates) if we're consistent in the code base.

I do understand your point that `exchange` provides more context to the expected party... would something like the below which addresses both concerns help ? 

> let exchange = customer
>
> controller exchange can